deploy/ansible/roles/iota-core-node/templates/docker-compose-iota-core.yml.j2

```diff
@@ -125,4 +117,26 @@ services:
     command: >
       --inx.address=iota-core:9029
       --faucet.bindAddress=0.0.0.0:8091
+{% endif %}
+
+{% if 'node-01' in inventory_hostname or 'node-02' in inventory_hostname or 'node-03' in inventory_hostname %}
+  inx-validator:
+    container_name: inx-validator
+    image: iotaledger/inx-validator:1.0-alpha
+    stop_grace_period: 1m
+    restart: unless-stopped
+    depends_on:
+      iota-core:
+        condition: service_healthy
```

Comment:
Isn't that some kind of race condition? Can the nodes even be healthy if there is no validator in the network yet?